src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/fix/leaked-passwords/LeakedPasswordsLayout.tsx

```diff
@@ -35,6 +35,7 @@ export interface LeakedPasswordsLayoutProps {
   subscriberEmails: string[];
   data: StepDeterminationData;
   isEligibleForPremium: boolean;
+  isPremium: boolean;
```

Comment:
Should this be `isPlus` or is Premium better since we are already committed to `isEligibleForPremium` on previous line?

Reply:
I made it `isPremiumUser` everywhere for consistency, we already have these in quite a few places so I think it'd be better to stay consistent for now (such as it is) and move everything from "Premium" to "Plus" together later.